src/storage.rs

```diff
@@ -264,7 +264,7 @@ impl Storage for MemStorage {
     }
 
     /// Implements the Storage trait.
```

Comment:
I've never seen this in impl blocks before, at first glance it seems totally redundant. Am i missing something?

Reply:
You're right. :)